src/ddmd/cppmangle.d

```diff
@@ -868,7 +872,11 @@ public:
             assert(t.basetype && t.basetype.ty == Tsarray);
             assert((cast(TypeSArray)t.basetype).dim);
             version (none)
-                buf.printf("Dv%llu_", (cast(TypeSArray *)t.basetype).dim.toInteger()); // -- Gnu ABI v.4
+            {
+                buf.writestring("Dv");
+                buf.print((cast(TypeSArray *)t.basetype).dim.toInteger()); // -- Gnu ABI v.4
+                buf.writeByte('_');
```

Comment:
Maybe turn `buf.print`/`writestring`/`writeByte` into a variable template to avoid multi-line function calls?